pkg/source/kyber-pmm/pool_tracker.go

```diff
@@ -41,17 +44,24 @@ func (t *PoolTracker) GetNewPoolState(
 
 	extra := Extra{}
 
-	priceLevels, err := t.getPriceLevelsForPool(ctx, p)
+	priceLevels, inventory, err := t.getPriceLevelsForPool(ctx, p)
 	if err != nil {
 		logger.WithFields(logger.Fields{
 			"poolAddress": p.Address,
 			"error":       err,
 		}).Errorf("failed to get price levels for pool")
 		return entity.Pool{}, err
 	}
+	//this is supposed to be big
+	p.Reserves = make([]string, 2)
```

Comment:
```suggestion
	p.Reserves = make([]string, len(p.Tokens))
```
I think it should be `len(p.Tokens)`?